docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,69 @@ helpviewer_keywords:
 
 <!-- For information about which features might be added in future releases, see the [Roadmap](https://github.com/dotnet/runtime/tree/81bf79fd9aa75305e55abe2f7e9ef3f60624a3a1/src/libraries/System.Text.Json/roadmap/README.md). [Restore this when the roadmap is updated.]-->
 
 Most of this article is about how to use the <xref:System.Text.Json.JsonSerializer> API, but it also includes guidance on how to use the <xref:System.Text.Json.JsonDocument> (which represents the Document Object Model or DOM), <xref:System.Text.Json.Utf8JsonReader>, and <xref:System.Text.Json.Utf8JsonWriter> types. The article is organized into sections in the following order:
 
+* [Table of differences between Newtonsoft.Json and System.Text.Json](#table-of-differences-between-newtonsoftjson-and-systemtextjson)
 * [Differences in **default** JsonSerializer behavior compared to Newtonsoft.Json](#differences-in-default-jsonserializer-behavior-compared-to-newtonsoftjson)
 * [Scenarios using JsonSerializer that require workarounds](#scenarios-using-jsonserializer-that-require-workarounds)
 * [Scenarios that JsonSerializer currently doesn't support](#scenarios-that-jsonserializer-currently-doesnt-support)
 * [JsonDocument and JsonElement compared to JToken (like JObject, JArray)](#jsondocument-and-jsonelement-compared-to-jtoken-like-jobject-jarray)
 * [Utf8JsonReader compared to JsonTextReader](#utf8jsonreader-compared-to-jsontextreader)
 * [Utf8JsonWriter compared to JsonTextWriter](#utf8jsonwriter-compared-to-jsontextwriter)
 
+## Table of differences between Newtonsoft.Json and System.Text.Json
+
+The following table lists `Newtonsoft.Json` features and `System.Text.Json` equivalents. The equivalents fall into the following categories:
+
+* Supported by built-in functionality. May require use of an attribute or global option in `System.Text.Json`.
+* Not supported, workaround is possible. The workarounds are [custom converters](system-text-json-converters-how-to.md) and may not provide 100% of the `Newtonsoft.Json` functionality. For some of these, sample code is provided.
+* Not supported, workaround is not practical.
+
+| Newtonsoft.Json feature                               | System.Text.Json equivalent |
+|-------------------------------------------------------|-----------------------------|
+| Case-insensitive deserialization by default           | ![check mark icon](../../../images/core/check-mark.png) [PropertyNameCaseInsensitive global setting](#case-insensitive-deserialization) |
```

Comment:
I think serializing/writing properties as camel case is another one that folks look for. This doc should mention that (probably as a separate heading).

Ordering of this section based on what might be most useful (we probably still want to keep the doc ordering structure though specific to things that are in different h2s - for instance JToken might be a common request but putting it in the middle of the serializer features seems odd):
Case-insensitive deserialization by default
[new camel casing section]
Minimal character escaping
NullValueHandling.Ignore
Allow comments
Allow trailing commas
Custom converter registration
No maximum depth by default
JToken (like JObject, JArray)
JsonTextReader
JsonTextWriter

In some of this re-ordering, we should consider restructuring them in the doc too so they show up in the same order. For instance, move minimal character escaping up in the doc too.